packages/caravan-fees/src/cpfp.ts

```diff
@@ -0,0 +1,105 @@
+import { Transaction } from "bitcoinjs-lib-v5";
+import { unsignedMultisigTransaction } from "@caravan/bitcoin";
+import BigNumber from "bignumber.js";
+import { CPFPOptions, UTXO } from "./types";
+import { estimateVirtualSize } from "./utils";
+
+/**
+ * Prepare a CPFP (Child-Pays-for-Parent) transaction.
+ * This function doesn't broadcast the transaction, it just prepares it.
```

Comment:
Maybe a description about what it means to "prepare" a transaction?

Reply:
Added in the earlier commit (here)[https://github.com/caravan-bitcoin/caravan/commit/4fce4fe2963337a6dcf56c153b80ab4fab1a5aa4#diff-852fd83c241ed78594dc29389ec04d1425d784e665c706434a8de046f7daa85dR18]